### PR TITLE
Setup kanban capstone

### PR DESCRIPTION
**Pull request by lu-jim**
First PR for this project
# Changes
- Chose an external API (https://www.themealdb.com/api.php)
- Set up the project with Webpack and Jest
- Chose our assets (logo, styling)
- Added header and footer

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  </nav>
+  <section class="container-fluid my-4 py-3">
+    <div class="row gx-5 justify-content-center">
+      <div class="col-3 mx-3 card d-flex align-content-around">
+        <img src="https://www.themealdb.com/images/media/meals/xvsurr1511719182.jpg" alt="Apple Crumble">
+        <h2 class="py-2">Apple & Blackberry Crumble</h2>
+        <input type="submit" name="comments" id="comments-0" value="Comments"
+          class="btn comment text-light btn-block my-2 ">
+        <input type="submit" name="reservations" id="reservation-0" value="Reservations"
+          class="btn reserve  text-light my-2 btn-block">
```
Comment (Emmy-github-webdev) [inline dist/index.html]: The Comment and Reservation buttons can be adjusted to look more beautiful

File: dist/index.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        </li>
+        <li class="nav-item mx-lg-2 mx-xl-3 mx-xxl-4">
+          <a class="nav-link" href="#ingredients" id="add-identifier">Ingredients</a>
+        </li>
+        <li class="nav-item ml-lg-2 ml-xl-3 ml-xxl-4">
+          <a class="nav-link" href="#about" id="contact-identifier">About</a>
+        </li>
+      </ul>
+    </div>
+  </nav>
```
Comment (Emmy-github-webdev) [inline dist/index.html:29]: image

The font looks smaller. Increasing the font will increase the visibility

File: dist/index.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+          class="btn reserve text-light my-2 btn-block">
+
+      </div>
+      <div class="col-3 mx-3 card d-flex align-content-around">
+        <img src="https://www.themealdb.com/images/media/meals/wxywrq1468235067.jpg" alt="">
+        <h2 class="py-2">Apple Frangipan Tart</h2>
+        <input type="submit" name="comments" id="comments-2" value="Comments"
+          class="btn comment text-light btn-block my-2">
+        <input type="submit" name="reservations" id="reservation-2" value="Reservations"
+          class="btn reserve text-light my-2 btn-block">
```
Comment (Emmy-github-webdev) [inline dist/index.html:67]: Add likes button to meet the requirement according to the design

image

**Review by Emmy-github-webdev — CHANGES_REQUESTED** [review]
@lu-jim,

Good job so far 👍 

You have done a good job. However, there are changes you need to make to move forward.
# Changes required ♻️ 

Please, check my comment below for the required changes.

Thank you 👏

File: dist/index.html (hunk 3 above)
Comment (lu-jim) [reply dist/index.html:67]: Done!

File: dist/index.html (hunk 2 above)
Comment (lu-jim) [reply dist/index.html:29]: Font size increased 👍

File: dist/index.html (hunk 1 above)
Comment (lu-jim) [reply dist/index.html]: Consider providing more concrete feedback so I can implement it 🙏

**Review by Emmy-github-webdev — APPROVED** [review]
Hi @lu-jim ,

Your project is complete! There is nothing else to say other than... it's time to merge it.

Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**